src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -138,38 +153,97 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
+        const maximumThreshold = 1; // one second threhsold
+        const timestamp = this.videoPlayer.current.getState().player.currentTime;
+        let seekTime: number = timestamp;
+        let nearestDifference = 2;
+        for (const child of this.props.childAssets) {
+            const childTimeDifference = Math.abs(child.timestamp - timestamp);
+            if (childTimeDifference < maximumThreshold && childTimeDifference < nearestDifference) {
+                nearestDifference = childTimeDifference;
+                seekTime = child.timestamp;
+            }
         }
+        if (seekTime !== timestamp) {
+            this.seekToTime(seekTime);
+        }
+
+        return seekTime !== timestamp;
     }
 
     /**
-     * Seeks the video to the timestamp set by the specified asset
-     * @param asset The child asset to seek to
+     * @name - Seek to timestamp
+     * @description - Moves the videos current position to the position passed in via the props
+     */
+    private seekToTimestamp = () => {
+        this.seekToTime(this.props.timestamp);
+    }
+
+    /**
+     * @name - Go to child asset
+     * @description - Moves the videos current position to a child asset
+     * @member asset - the asset to move the position to
      */
     private goToChildAsset(asset: IAsset) {
-        this.videoPlayer.current.pause();
-        this.videoPlayer.current.seek(asset.timestamp);
+        this.seekToTime(asset.timestamp);
     }
 
     /**
-     * Wired up to the underlying video "Player" component and receive notifications for all video state changes
-     * @param state The current state of the video player component
-     * @param state The previous state of the video player component
+     * @name - Seek to time
+     * @description - Seeks the current video to the passed in time stamp, pausing the video before hand
+     * @member seekTime - Time (in seconds) in the video to seek to
      */
+    private seekToTime(seekTime: number) {
+        if (seekTime > 0) {
+            // Before seeking, pause the video
+            if (!this.videoPlayer.current.getState().player.paused) {
+                this.videoPlayer.current.pause();
+            }
+            this.videoPlayer.current.seek(seekTime);
+        }
+    }
+
     private onVideoStateChange = (state: Readonly<IVideoPlayerState>, prev: Readonly<IVideoPlayerState>) => {
         if (!this.state.loaded && state.readyState === 4 && state.readyState !== prev.readyState) {
             // Video initial load complete
             this.raiseLoaded();
             this.raiseActivated();
             this.seekToTimestamp();
         } else if (state.paused && (state.currentTime !== prev.currentTime || state.seeking !== prev.seeking)) {
-            // Video is paused
-            this.raiseChildAssetSelected(state);
+            // Video is paused, first try to move to a nearby frame if one exists
+            if (!this.moveToNearestChildFrame()) {
```

Comment:
I'm curious to the end user experience of this change... If i'm attempting to manually scrub in a video to find the right frame to tag and I'm close to a previous tagged/visited frame it would automatically skip me back there?

Reply:
I argued with myself a lot about this. Without it, it's near impossible to sync to the "right" spot, but it also removes the ability for super fine grained scrubbing too. Maybe if the click handlers work that solves the problem.

Reply:
Removed since the div clicks seem to solve the problem this was intending, although this still can be a bit weird if you stop very near an already existing child asset.